src/commands/deploy/deploy.types.ts

```diff
@@ -14,14 +14,7 @@ export type DeployTokenRegistryCommand = NetworkAndWalletSignerOption &
     registryName: string;
     registrySymbol: string;
     passedOnWallet?: Wallet | ConnectedSigner;
-  };
-
-export type DeployTitleEscrowCreatorCommand = NetworkAndWalletSignerOption & GasOption;
-
-export type DeployTitleEscrowCommand = NetworkAndWalletSignerOption &
-  GasOption & {
-    tokenRegistry: string;
-    beneficiary: string;
-    holder: string;
-    titleEscrowFactory?: string;
+    factoryAddress?: string;
+    tokenImplementationAddress?: string;
+    deployerAddress?: string;
```

Comment:
(qn) i see that these will fallback to the hardcoded values as well, can help me understand the use of these other 2 params?